recipes-images/images/engicam-gwc.bb

```diff
@@ -105,6 +105,9 @@ IMAGE_INSTALL:append = " \
 	libedit \
 	gpsd \
 	libgps \
+        libestr \
+        libfastjson \
+	rsyslog \
```

Comment:
If you implement the below suggestion to put `VIRTUAL-RUNTIME_base-utils-syslog` in system configuration then you should not need to explicitly mention it here anymore. But you could as well leave it still just to make it more explicit for future readers.